packages/visx-xychart/src/components/series/private/AnimatedBars.tsx

```diff
@@ -65,6 +66,7 @@ export default function AnimatedBars<XScale extends AxisScale, YScale extends Ax
         item == null || key == null ? null : (
           <animated.rect
             key={key}
+            tabIndex={isFocusable ? 0 : undefined}
```

Comment:
noting that this implementation requires the `SVG 2.0` spec – [`SVG 1.1` doesn't support `tabIndex`](https://allyjs.io/tutorials/focusing-in-svg.html) which means this may not work on older browsers. the alternative approach (used in `data-ui`) was to wrap each element in an `<a />` element, which are consistently tab-able by all browsers.

For now I think this more modern approach is more ideal since it's simpler and [it is supported across all modern browsers](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/tabindex)
![image](https://user-images.githubusercontent.com/4496521/101396241-e2ae2e00-387f-11eb-851f-addc8b69541a.png)